ftdc/cmd/parser.go

```diff
@@ -160,6 +210,13 @@ var ratioMetricToFields = map[string]ratioMetric{
 	"RxPacketsPerSec":        {"RxPackets", ""},
 	"TxBytesPerSec":          {"TxBytes", ""},
 	"RxBytesPerSec":          {"RxBytes", ""},
+
+	// Dan: Just tacking these on -- omitted metrics from this list does not mean they shouldn't* be
+	// here. Also, personally, sometimes I think not* doing PerSec for these can also be
+	// useful. Maybe we should consider including both the raw and rate graphs. Instead of replacing
+	// the raw values with a rate graph.
+	"GetImagePerSec":    {"GetImage", ""},
+	"GetReadingsPerSec": {"GetReadings", ""},
```

Comment:
Maybe I missed this, but do we now have stats for every gRPC method invoked?

Reply:
We do! For unary calls at least.